src/digitalmarketplace/all.js

```diff
@@ -7,11 +8,16 @@ function initAll (options) {
   if ($cookieBanner) {
     new CookieBanner($cookieBanner).init()
   }
+  const $cookieSettingsPage = document.querySelector('[data-module="dm-cookie-settings"]')
+  if ($cookieSettingsPage) {
+    new CookieSettings($cookieSettingsPage).init()
```

Comment:
We could pass the cookie banner object to the CookieSettings object here

Reply:
I'll use `document.querySelector` as we're already doing so for showing/hiding error messages.

Reply:
Did you decide against the idea of just not including the banner in the template?

Reply:
I'm going to do both - belt and braces.